components/_loader.scss

```diff
@@ -0,0 +1,68 @@
+// This is convoluted because the markup
+// doesn't make it easy to only show the header
+// We want to hide the buttons and the list of cities
+.loader {
+  display: none;
+}
+
+.cities-partial.loading {
+  .loader { display: block; }
+  .new-city-cta, .cities-lists, .header-small { display: none !important; }
```

Comment:
!important should not be used<br>Each selector in a comma sequence should be on its own line